GCRCatalogs/alphaq.py

```diff
@@ -68,8 +68,8 @@ def _subclass_init(self, filename, **kwargs):
             'stellar_mass':       'totalMassStellar',
             'size_disk_true':     'morphology/diskHalfLightRadiusArcsec',
             'size_bulge_true':    'morphology/spheroidHalfLightRadiusArcsec',
-            'disk_Sersic_index':  'morphology/diskSersicIndex',
-            'bulge_Sersic_index': 'morphology/spheroidSersicIndex',
+            'disk_Sersic':  'morphology/diskSersicIndex',
+            'bulge_Sersic': 'morphology/spheroidSersicIndex',
```

Comment:
should be `sersic_bulge` (lower case, component name should come after quantity name)